test/fmt.jl

```diff
@@ -20,8 +20,19 @@ i = 1234567
 @test fmt(i) == "1234567"
 @test fmt(i,:commas) == "1,234,567"
 
-@test_throws ErrorException fmt_default(Real)
-@test_throws ErrorException fmt_default(Complex)
+@test fmt(2 - 3im, 10) == "   2 - 3im"
+@test fmt(pi - 3im, 15, 2) == "  3.14 - 3.00im"
+
+@test fmt(3//4, 10) == "      3//4"
+@test fmt(1//2 + 6//2 * im, 15) == " 1//2 + 3//1*im"
+
+fmt_default!(Rational, 'f', prec = 2)
+fmt_default!(Format.ComplexRational, 'f', prec = 2)
```

Comment:
I think this is broken now - there is no `Format.ComplexRational` any more.

Reply:
`fmt_default!(Complex{T} where T<:Rational, 'f', prec = 2)` fails. It seems that internally the typevars `T` in `src/fmt.jl` and `test/fmt.jl` are not identical. Hence, I am going back to defining a type constant.

Reply:
done